### DISCUSSION TITLE
posts/2023/08/liu-shui-zhang/

### DISCUSSION THREAD

**Discussion by giscus** [residualsun1/Residualsun#17] [General]
# posts/2023/08/liu-shui-zhang/

https://guozheng.rbind.io/posts/2023/08/liu-shui-zhang/

**Comment by CyrusYip** [comment]
我发现你会发布一篇日记后再添加新内容，例如：[黑石屿记忆（五）](https://github.com/residualsun1/Residualsun/commits/main/content/posts/2023/07/naozhou-fifth-day.md)，这样子读者看了第一次发的之后就不知道后面更新过了，除非订阅了[博客源代码的历史](https://github.com/residualsun1/Residualsun/commits/main.atom)。我建议写完整再发出来，不用急着每天都发，放在笔记软件改几天再发也是可以的。我好像还看过一两次只有标题的文章，这会让读者扑个空。

写作要在「写完就发」和「追求完美」之间取得平衡，前者让读者看到未经打磨的文章，后者让作者拖稿到一直不发布。

**Reply by residualsun1** [reply]
> 我建议写完整再发出来，不用急着每天都发，放在笔记软件改几天再发也是可以的。

接受建议^^

---
另，咋现在都没见你更新博客了？

**Reply by CyrusYip** [reply]
太忙啦，所以没怎么更新。你也可以用每写一篇文章就建一个 git 分支，写完合并到主分支，用 `git rebase` 可以把多个 commit 合并成一个。
